docs/source/installation.mdx

```diff
@@ -0,0 +1,56 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Installation
+
+There is currently no `pip` support for `optimum-nvidia` as transitive dependencies are missing on the PyPi index.
+The easiest ways to get started are either:
+- Pulling prebuilt docker container `huggingface/optimum-nvidia`
+- Building the docker container locally
+
+
+## Pulling prebuilt Docker Container
+
+Hugging Face pushes and hosts versions of the container matching the release version of `optimum-nvidia` on the Docker hub.
+This container comes with all the required dependencies to execute `float32`, `float16`, `int8` compressed or quantized and `float8` models.
+
+To get started, simply pull the container with the following command:
+```bash
+docker run -it --gpus all --ipc host huggingface/optimum-nvidia
+```
+
+## Building Docker Container locally
+
+If you want to build your own image and/or customize it, you can do so by using the three steps process described below:
+
+1. Clone `optimum-nvidia` repository `git clone --recursive --depth=1 https://github.com/huggingface/optimum-nvidia`
```

Comment:
```suggestion
1. Clone `optimum-nvidia` repository:
```bash
git clone --recursive --depth=1 https://github.com/huggingface/optimum-nvidia && cd optimum-nvidia
```
```

Reply:
Not sure if the formatting got thrown off here, but I'd suggest making this a copy-able code snippet